### ISSUE TITLE
Feet muscle atrophy

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11499]
**Preferred term label:**

**Synonyms**
Atrophy of muscles in feet
Amyotrophy involving the musculature of the feet
Feet muscle wasting 

**Definition (free text, please give PubMed ID)**
PMID: 36108117  **Atrophy of the muscles in the hands and feet** has been reported in 4% of affected individuals, although this feature was not assessed by Green et al [2020] in their cohort of 20 affected individuals. It is unclear whether this is a characteristic feature of TNXB-related clEDS.

PMID: 28306229 and PMID: 32572181 The major clinical criteria of clEDS are (1) skin hyper extensibility, with velvety skin texture and absence of atrophic scarring; (2) generalized joint hypermobility with or without recurrent dislocations (most commonly shoulder and ankle); and (3) easy or spontaneous bruising of skin. 
Minor criteria are (1) foot deformities including broad/plump forefoot, brachydactyly with excessive skin, pes planus, hallux valgus, and piezogenic papules; (2) edema in the legs in absence of cardiac failure; (3) mild proximal and distal muscle weakness; (4) axonal polyneuropathy; (5) **atrophy of** **muscles in hands and feet**; (6) acrogeric hands, mallet finger(s), clinodactyly, and brachydactyly; and (7) vaginal/uterus/rectal prolapse

**Parent term (use hpo.jax.org/app)**
Distal lower limb amyotrophy  HP:0008944

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Classical-like Ehlers-Danlos syndrome type 1 ORPHA:230839

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f  I think this should be Foot Small Muscle Atrophy because the larger muscles are partially outside the foot (extrinsic) and probably need separate terms.

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001485
- label: Foot small muscle atrophy
- parents: [Distal lower limb amyotrophy (HP:0008944)](https://hpo.jax.org/app/browse/term/HP:0008944)
- pmid: PMID:15920063
- def: Wasting of the small muscles of the foot such as the abductor hallucis, flexor digitorum brevis, and the interossei muscles.
- comment: This sign may indicate diabetix neuropathy of length-dependent polyneuropathies such as Charcot-Marie-Tooth disease.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11499
- synonyms:
